project_hr/tests/test_project_hr.py

```diff
@@ -59,6 +59,7 @@ def test_task(self):
             'user_id': self.user_2.id,
             'category_ids': [(6, 0, self.hr_category.ids)],
         })
+        self.task._compute_allowed_user_ids()
```

Comment:
This mustn't be called manually. If you need to do this, something in the depends is not correct. Other option is that certain operations must call a cache invalidation.